stylesheets/components/badges/badges.css

```diff
@@ -0,0 +1,38 @@
+.badge {
+    background-color: #34D298;
+    position: absolute;
+    top: 0;
+    right: 0;
+    border-style: solid;
+    border-color: var(--color-white);
+    border-radius: var(--br-full);
+}
+
+.badge-container {
+    position: relative;
+    width: max-content;
+}
```

Comment:
You can move this above the badge styles as .badge-container is the parent to .badge, so this would make more sense.

Reply:
Good point. Addressed.